electrumx/server/env.py

```diff
@@ -99,6 +101,8 @@ def __init__(self, coin=None):
         if {service.protocol for service in self.services}.intersection(self.SSL_PROTOCOLS):
             self.ssl_certfile = self.required('SSL_CERTFILE')
             self.ssl_keyfile = self.required('SSL_KEYFILE')
+        if any([service.protocol == 'bolt8' for service in self.services]):
+            self.bolt8_keyfile = self.required('BOLT8_KEYFILE')
```

Comment:
Maybe make this config key optional (and put it inside DB_DIRECTORY by default.
Regardless, the envvar should be documented.